setup.py

```diff
@@ -32,5 +32,5 @@
             'mbed-cli=mbed.mbed:main',
         ]
     },
-    python_requires='>=2.7.10, <3',
+    python_requires='>=2.7.10',
```

Comment:
Perhaps this change should come with the other PR.

Reply:
Agreed. Will revert this.